charts/datadog/values.yaml

```diff
@@ -1506,3 +1506,128 @@ providers:
       ## IMDS metadata endpoint is not accesible to containers. This flag mounts the host's
       ## `/var/lib/cloud/data/instance-id` and uses that for Agent's hostname instead.
       useHostnameFromFile: false
+
+observabilityPipelines:
```

Comment:
IMO we should not put the full values file in from vector into the datadog chart file. It will be very difficult to maintains.

You can look at what we did for the kube-state-metrics chart dependencies

Reply:
This is just overriding a few options from the full values file, just the stuff that's needed to properly wire up to the Agent.